src/graphql/schema.js

```diff
@@ -301,6 +301,7 @@ input WithdrawFilter {
 input TransactionFilter {
   eventAddress: String
   transactorAddress: String
+  txid: String
```

Comment:
create a new `PendingTransactionsFilter` since we dont use txid in the `Transactions` query.